modular_skyrat/master_files/code/modules/antagonists/_common/objectives.dm

```diff
@@ -1,13 +1,13 @@
 /// Chance that the traitor could roll hijack if the pop limit is met.
-#define HIJACK_PROB 10
+#define HIJACK_PROB 0
 /// Hijack is unavailable as a random objective below this player count.
 #define HIJACK_MIN_PLAYERS 50
 
 /// Chance the traitor gets a martyr objective instead of having to escape alive, as long as all the objectives are martyr compatible.
 #define MARTYR_PROB 20
 
 /// Chance the traitor gets a kill objective. If this prob fails, they will get a steal objective instead.
-#define KILL_PROB 50
+#define KILL_PROB 0
```

Comment:
Sure to both of these, though they probably apply to non-progtot

Reply:
they do and thats intentional as well.

Reply:
Hard disagree. Steal objectives are boring as fuck.
Removing kill objectives removes all reason for antagonists to harm people, and as such, makes them forcefully stealth, which makes for extremely fucking boring rounds.
It also brings the server 2 steps closer to Skyrat where all antagonism is actively hated and it's a giant cuddlepile hugbox.

Reply:
Lowering kill prob is OK, removing kill objectives is lame.

Reply:
Agreed!

Reply:
> 

If you need to steal an heirloom from someone as an objective, and they refuse to give it to you, you can kill them, same with kidnapping people. All this change intends to do is turn killing from killing for literally no reason what so ever, into killing for a reason. Killing is good, killing for the SAKE of killing is bad.

Reply:
> State what the PR does in full, or strip out the 'unintended' replocks.

enabling progtot means enabling replocks, its intended.

Reply:
> All this change intends to do is turn killing from killing for literally no reason what so ever, into killing for a reason.

We had reasons before. You took them away. They were called kill objectives.